pyasn/mrtx.py

```diff
@@ -109,6 +117,15 @@ def dump_prefixes_to_text_file(ipasn_data, out_text_file_name, orig_mrt_name, de
         fw = open(out_text_file_name, 'wt')
     else:
         fw = open(out_text_file_name, 'wt', encoding='ASCII')
+
+    text_prefixes = dump_prefixes_to_text(ipasn_data, orig_mrt_name, debug_write_sets=debug_write_sets)

```

Comment:
Good point, we are not passing fw, because we want the result of ```dump_prefixes_to_text``` to be ```str```, as this string can more easily be passed  around elsewhere.